packages/use-contractkit/src/utils/helpers.ts

```diff
@@ -90,12 +90,14 @@ export const loadPreviousConfig = (
 };
 
 export function clearPreviousConfig(): void {
-  Object.values(localStorageKeys).forEach((val) =>
-    localStorage.removeItem(val)
-  );
+  Object.values(localStorageKeys).forEach((val) => {
+    if (val === localStorageKeys.lastUsedWalletId) return;
```

Comment:
Why aren't we wanting to remove these things?

Reply:
We now have a `recent` wallet in the wallets "tray". So I'm using that key to identity the most recently used wallet.